src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -7646,7 +7647,7 @@ public unsafe bool TryHPackAppend(int index, ReadOnlySpan<byte> value)
                 }
 
                 // We didn't have a previous matching header value, or have already added a header, so get the string for this value.
-                var valueStr = value.GetRequestHeaderString(nameStr, EncodingSelector);
+                var valueStr = value.GetRequestHeaderString(nameStr, EncodingSelector, checkForNewlineChars);
```

Comment:
This can be inlined, no need for the variable.
```suggestion
                var valueStr = value.GetRequestHeaderString(nameStr, EncodingSelector, checkForNewlineChars: true);
```

Reply:
Not exactly, this same line is used to generate `Append` and `TryHPackAppend`, and it won't always be true in `Append`. I'm changing the source it's generated from.

Reply:
Not sure that works either actually, since the value that `Append` inserts into the `AppendValue` string is based off of the parameters it was called with, which isn't known at the time we generate the file